crates/testing-utils/eigenlayer/src/env.rs

```diff
@@ -11,6 +11,7 @@ use eigensdk::utils::slashing::middleware::registry_coordinator::RegistryCoordin
 use url::Url;
 
 // ================= Core Eigenlayer Deployment Addresses =================
+// TODO(daniel): update these addresses to the new addresses of Tangle testnet
```

Comment:
These have nothing to do with Tangle testnet, we care only about anvil local testnet with forked state, the live testnet eigenlayer uses, and mainnet where eigenlayer is deployed. None of these are Tangle.